conda_forge_tick/migrators/nvtools.py

```diff
@@ -241,7 +211,40 @@ def pr_body(
             The body of the PR message
             :param feedstock_ctx:
         """
-        body = f"{AddNVIDIATools.__doc__}\n\n"
+        body = """\
+In order to ensure that NVIDIA's redistributed binaries (redists) are being packaged
+correctly, NVIDIA has created a package containing a collection of tools to perform
+common actions for NVIDIA recipes.
+
+At this time, the package may be used to check Linux binaries for their minimum glibc
+requirement in order to ensure that the correct metadata is being used in the conda
+package.
+
+This migrator will attempt to add this glibc check to all feedstocks which download any
+artifacts from https://developer.download.nvidia.com. The check involves adding
+"cf-nvidia-tools" to the top-level build requirements and something like:
+
+```bash
+check-glibc "$PREFIX"/lib/*.so.* "$PREFIX"/bin/*
+```
+
+to the build script after the package artifacts have been installed.
+
+> [!NOTE]
+> A human needs to verify that the glob expression is checking all of the correct
+> artifacts!
+
+> [!NOTE]
+> If the recipe does not have a top-level requirements.build section, it should be
+> refactored so that the top-level package does not share a name with one of the
+> outputs. i.e. The top-level package name should be something like "libcufoo-split".
+
+More information about cf-nvidia-tools is available in the feedstock's [README](https://github.com/conda-forge/cf-nvidia-tools-feedstock/tree/main/recipe).
+
+Please ping carterbox for questions.
```

Comment:
As a future note let's also link the main issue so it is easier to track all the migration PRs that are made and see when they are closed out

```suggestion
Please ping carterbox for questions.

xref: https://github.com/conda-forge/cuda-feedstock/issues/62
```

Reply:
That's a good point, since the migrations tracking page doesn't update immediately.